sklearn/metrics/_regression.py

```diff
@@ -480,6 +485,8 @@ def mean_squared_log_error(
     >>> y_pred = [2.5, 5, 4, 8]
     >>> mean_squared_log_error(y_true, y_pred)
     0.039...
+    >>> mean_squared_log_error(y_true, y_pred, squared=False)
+    0.197...
```

Comment:
This is making the doctest fail. Are you sure of the value?

Reply:
Ops! Sorry it was 0.199... and I had written 0.197.. My bad :(